edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/DeviceConnectionMetrics.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core
+{
+    using System.Collections.Generic;
+    using Microsoft.Azure.Devices.Edge.Util.Metrics;
+    using EdgeMetrics = Util.Metrics.Metrics;
+
+    public static class DeviceConnectionMetrics
+    {
+        static readonly List<string> EmptyStringList = new List<string>();
+        static readonly string[] EmptyStringArray = new string[0];
```

Comment:
please consider using `Array.Empty<string>()`

Reply:
Will do. There is a interesting article [here](https://www.tabsoverspaces.com/233833-best-way-to-create-an-empty-collection-array-and-list-in-csharp-net) about performance.